# UiLayoutState Message

std_msgs/Header header

std_msgs/UInt64 activeApp
std_msgs/Bool sidebarCollapsed
std_msgs/Bool mapEnabled
std_msgs/Bool mockEngaged



# struct UiLayoutState {
#   activeApp @0 :App;
#   sidebarCollapsed @1 :Bool;
#   mapEnabled @2 :Bool;
#   mockEngaged @3 :Bool;

#   enum App {
#     home @0;
#     music @1;
#     nav @2;
#     settings @3;
#     none @4;
#   }
# }
